core/controllers/questions_list.py

```diff
@@ -47,7 +47,12 @@ class QuestionsListHandler(base.BaseHandler):
     @acl_decorators.open_access
     def get(self, comma_separated_skill_ids):
         """Handles GET requests."""
-        start_cursor = self.request.get('cursor')
+        more = True
+        offset = self.request.get('offset')
+        if offset == '':
```

Comment:
This case should not be possible. Please verify that no caller passes an empty string in.

If the offset is not a number then this handler should error with 400.

Reply:
@seanlip URLs like `questions_list_handler/1?offset=` and `questions_list_handler/1` have offset value as an empty string. I don't think an error should be thrown for these URLs. What do you think? Thanks!

Reply:
We have control over those URLs. So, we should fix such instances in the frontend such that the incorrectly-formatted URLs never get called in the backend. (In other words, this still needs to be fixed.)

Reply:
Done